### ISSUE TITLE
Why can you delete a ship if you cant add a ship without buying one ingame?

### ISSUE THREAD

**Issue by TeKett** [goatfungus/NMSSaveEditor#901]
Question that should have been better apt for the discussion page, but you dont have one.

**Comment by cjg67** [comment]
Because people have ships in a save and perhaps wish to delete them.
There is also a seed generator that you can use to generate ships if you have the correct ship seed.

**Comment by socrlax24** [comment]
I'm pretty sure the OP is asking why you can't ADD ships via the editor if you're able to REMOVE ships via the editor. I don't think they're requesting the ability to remove be ... removed.

**Comment by tarasis** [comment]
Indeed, I'd love to be able to add a ship. Even more relevant with the new expansion released today.
